torchvision/models/inception.py

```diff
@@ -61,134 +61,35 @@ def inception_v3(pretrained=False, progress=True, **kwargs):
     return Inception3(**kwargs)
 
 
-class Inception3(nn.Module):
```

Comment:
Can you revert all this code-movement?

Reply:
The code movement is necessary as the reason above.